clr-k8s-examples/tests/e2e/run_e2e.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+# Runs upstream k8s e2e tests against existing cloud native basic cluster
+# Requires cluster to already be up
+# To specify a parameter for --ginkgo.focus as described below, provide a focus as the first argument to this script
+#     https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/e2e-tests.md#building-kubernetes-and-running-the-tests
+# One example would be Feature:Performance. The script will add square brackets for you
+# For other examples of values, see
+#     https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/e2e-tests.md#kinds-of-tests
+
+if [ ! -z $1 ]
+then
+    FOCUS=$1
+    echo Running e2e tests where spec matches $1
+else
+    echo Running all e2e tests, this will take a long time
+fi
+
+sudo swupd bundle-add go-basic
+
+go get -d k8s.io/kubernetes
+go get -u k8s.io/test-infra/kubetest
+
```

Comment:
Moved the error on exit options below go get calls as even a successful get of k8s.io/kubernetes results in an error message stating that no go files were found.  However, the repo seems to have been pulled fine and tests run.

Reply:
Does the script fail cause of the message you are getting due to `go get`?

Reply:
Yes line 21 returns an error even on successful get.  I was researching go get options to ignore the message of no go files found

Reply:
Using git to get kubernetes instead